src/loguru_parallel/enqueue.py

```diff
@@ -15,22 +14,19 @@ def create_log_queue() -> Queue:
 
 
 def enqueue_logger(queue: Queue) -> None:
-    """Configure the current process's logger to enqueue records.
+    """Configure the loguru.logger instance to send logs to a queue.
 
-    This means that all handlers will be removed and replaced with a single handler that
-    enqueues records to the global log queue.
+    This means that all handlers are deleted and replaced with a single handler that
+    enqueues records log queue passed to this function.
     """
-    _queue = queue
-    if not logger._core.handlers:
-        logger.add(lambda dummy: dummy)
-    handlers = logger._core.handlers
-    for handler in handlers.values():
-        state = handler.__getstate__()
-        sink = CallableSink(lambda record: _queue.put(record))
-        sink._enqueued = True
-        state["_sink"] = sink
-        state["_name"] = getattr(sink, "__name__", None) or repr(sink)
-        handler.__setstate__(state)
+    logger.remove()
+
+    def queue_sink(record):
+        queue.put(record)
+
+    logger.add(queue_sink)
+    handler = list(logger._core.handlers.values())[0]
```

Comment:
What happens if we add a handler after calling enqueue_logger? Do we not expect to add it?

Reply:
Great question!
Yes, the expectation is that no handlers are added after calling  `enqueue_logger`.
Handlers could be added to the `_sink_logger` of LoguruQueueListener. Which means I should probably make this a publicly accessible attribute in the future 🤔 .

Reply:
I see. I understand.
Hmm, even if we provide a way to add handlers by publishing them, I'm concerned about whether it will be possible to properly distinguish and use the existing loguru logger instances and LoguruQueueListener logger instances. For example, if there are loguru.logger.add and loguru.logger.remove in the existing code, I don't know how to modify it so that loguru-parallel can be used. I think it's fine to have a policy that prohibits that kind of usage for now, but I'd like to be able to use them together in the future.